src/utils.js

```diff
@@ -187,7 +194,7 @@ export function combineSpacing({
   }
 
   const flatProps = {
-    ...props,
+    ...replaceAliasValues(props, spacingAliases),
```

Comment:
This fixes a bug where aliases for individual values (e.g. `marginBottom`) were not replaced

Reply:
Is there a test that covers this fix?

Reply:
No, there isn't but there should be, will update